src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -332,6 +332,7 @@ def register_common_storage_account_options(context):
               **model_choice_list(ResourceType.MGMT_STORAGE, 'Kind'))
     c.reg_arg('tags', tags_type)
     c.reg_arg('custom_domain', help='User domain assigned to the storage account. Name is the CNAME source.')
+    c.reg_arg('sku', help='The storage account SKU.', default=enum_default(ResourceType.MGMT_STORAGE, 'SkuName', 'standard_ragrs'), **model_choice_list(ResourceType.MGMT_STORAGE, 'SkuName'))
```

Comment:
Can you break the line.

Reply:
Please type `-h` and paste the output here.

Reply:
--sku                         : The storage account SKU.  Allowed values: Premium_LRS,
                                    Standard_GRS, Standard_LRS, Standard_RAGRS, Standard_ZRS.
                                    Default: Standard_RAGRS.